crates/apollo-compiler/tests/validation/types.rs

```diff
@@ -2018,34 +2016,3 @@ mod variable_default_values {
         );
     }
 }
-
-#[test]
-fn handles_built_in_type_redefinition() {
```

Comment:
This `types.rs` file was for ValuesOfCorrectType Rule tests, so I moved these builtin_type_redefinition tests into their own file.